Appraisals

```diff
@@ -1,21 +1,31 @@
 # frozen_string_literal: true
 
 appraise "rails_4_2" do
-  version = "~> 4.2.0"
-  gem "activerecord", version
+  gem "rails", "~> 4.2.0"
+  gem "sqlite3", "~> 1.3.0"
 end
 
 appraise "rails_5_0" do
-  version = "~> 5.0.0"
-  gem "activerecord", version
+  gem "rails", "~> 5.0.0"
+  gem "sqlite3", "~> 1.3.0"
 end
 
 appraise "rails_5_1" do
-  version = "~> 5.1.0"
-  gem "activerecord", version
+  gem "rails", "~> 5.1.0"
+  gem "sqlite3", "~> 1.4.0"
 end
 
 appraise "rails_5_2" do
-  version = "~> 5.2.0"
-  gem "activerecord", version
+  gem "rails", "~> 5.2.0"
+  gem "sqlite3", "~> 1.4.0"
+end
+
+appraise "rails_6_0" do
+  gem "rails", "~> 6.0.0.rc1"
```

Comment:
Would this lock rails at `6.0.0` though?

Reply:
Not sure, I'm going to bump the version as soon as next RC or release comes in

Reply:
Seems fine :P
```ruby
Gem::Requirement.create("~> 6.0.0.rc1").satisfied_by?(Gem::Version.create("6.0.0"))
#=> true
Gem::Requirement.create("~> 6.0.0.rc1").satisfied_by?(Gem::Version.create("6.0.2"))
#=> true
Gem::Requirement.create("~> 6.0.0.rc1").satisfied_by?(Gem::Version.create("6.1.0"))
#=> false
```